tools/remove_local_ironic.sh

```diff
@@ -3,7 +3,8 @@
 set -xe
 
 # This script removes local ironic containers. 
-# It requires ${CONTAINER_RUNTIME} variable to be defined first
+
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
 
 for name in ironic ironic-api ironic-conductor ironic-inspector dnsmasq httpd mariadb ipa-downloader \
```

Comment:
Are you planning to keep these for now and once everything is working as expected remove `ironic-api` && `ironic-conductor` from this list?

Reply:
Yeah, probably keep for a while until we make sure all local environments are updated.